ssz/sedes/base.py

```diff
@@ -0,0 +1,149 @@
+import abc
```

Comment:
As you use `@abc.abstractmethod` a lot, maybe just import it.

```suggestion
from abc import (
    ABC,
    abstractmethod,
)
```